notebooks/224-stable-diffusion-text-to-image/224-stable_dffusion-text-to-image.ipynb

```diff
@@ -0,0 +1,1271 @@
+        "            if isinstance(self.scheduler, LMSDiscreteScheduler):\n",
+        "                sigma = self.scheduler.sigmas[i]\n",
+        "                # the model input needs to be scaled to match the continuous ODE formulation in K-LMS\n",
+        "                latent_model_input = latent_model_input / ((sigma**2 + 1) ** 0.5)\n",
+        "\n",
+        "            # predict the noise residual\n",
+        "            noise_pred = self.unet([latent_model_input, t, text_embeddings])[0]\n",
+        "            # perform guidance\n",
+        "            if do_classifier_free_guidance:\n",
+        "                noise_pred_uncond, noise_pred_text = noise_pred[0], noise_pred[1]\n",
+        "                noise_pred = noise_pred_uncond + guidance_scale * (noise_pred_text - noise_pred_uncond)\n",
+        "\n",
+        "            # compute the previous noisy sample x_t -> x_t-1\n",
+        "            if isinstance(self.scheduler, LMSDiscreteScheduler):\n",
+        "                latents = self.scheduler.step(noise_pred, i, latents, **extra_step_kwargs)[\"prev_sample\"]\n",
+        "            else:\n",
+        "                latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs)[\"prev_sample\"]\n",
+        "            if gif:\n",
+        "                image = self.vae( 1 / 0.18215 * latents)[0]\n",
+        "                image = (image / 2 + 0.5).clip(0, 1)\n",
+        "                image = image.transpose(0, 2, 3, 1)\n",
+        "                if output_type == \"pil\":\n",
+        "                    image = self.numpy_to_pil(image)\n",
+        "                img_buffer.extend(image)\n",
+        "\n",
+        "\n",
+        "        # scale and decode the image latents with vae\n",
+        "        latents = 1 / 0.18215 * latents\n",
+        "        image = self.vae(latents)[0]\n",
+        "\n",
+        "        image = (image / 2 + 0.5).clip(0, 1)\n",
+        "        image = image.transpose(0, 2, 3, 1)\n",
+        "        if output_type == \"pil\":\n",
+        "            image = self.numpy_to_pil(image)\n",
+        "\n",
+        "        return {\"sample\": image, 'iterations': img_buffer}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Configure Inference Pipeline"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "First of all, we should create instance of our OpenVINOModel wrapper. We use it for more convinient getting inference results."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from openvino.runtime import Core\n",
+        "core = Core()\n",
+        "text_enc = OpenVINOModel(core, 'text_encoder.xml', device='CPU')\n"
```

Comment:
I didn't get why do we need `OpenVINOModel`? It brings an additional burden to understand this from the people who are unfamiliar with OpenVINO. Especially, if note that there is a class `openvino.runtime.Model` which does the same. If we need a benchmarking it is more demonstrative to do it inline in the code.

Reply:
BTW, `OpenVINOStableDiffusionPipeline` is ok here. You can just call it like `OVStableDiffusionPipeline` like we are calling it in the HF Optimum.

Reply:
> I didn't get why do we need OpenVINOModel? It brings an additional burden to understand this from the people who are unfamiliar with OpenVINO. Especially, if note that there is a class openvino.runtime.Model which does the same. If we need a benchmarking it is more demonstrative to do it inline in the code.

The initial idea was to just create wrapper to use openvino model in regular stable diffusion pipeline, but I decided to create standalone pipeline, these wrappers hide outputs getting logic, which is not looks pretty in OV model, so I rewrite pipeline without additional wrappers